Add prod run for configuration (#5041)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/govern/project-dependencies.md

In order to add project dependencies and resolve cross-project `ref`, you must:
- Use dbt v1.6 or higher for **both** the upstream ("producer") project and the downstream ("consumer") project.

> - Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). Once defined, re-run a production job to apply the change.

Reviewer: ```suggestion
- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). To apply the change, rerun a production job.
```

In order to add project dependencies and resolve cross-project `ref`, you must:
- Use dbt v1.6 or higher for **both** the upstream ("producer") project and the downstream ("consumer") project.

> - Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access)

Author: ```suggestion
- Define models in an upstream ("producer") project that are configured with [`access: public`](/reference/resource-configs/access). Once defined, re-run a production job to apply the change.
```

>   - Once defined, re-run a production job to apply the change.

Author: ```suggestion
```

>   - Once defined, re-run a production job in order to apply the change.

Author: ```suggestion
  - Once defined, re-run a production job to apply the change.
```
